internal/app/components/terms/terms_test.go

```diff
@@ -5,7 +5,7 @@ import (
 	"testing"
 )
 
-func TestNewTerm(t *testing.T) {
+func Test_NewTerm(t *testing.T) {
```

Comment:
I think `TestNewTerm` is the right name, if it's just a function it's just `Test` + `FuncName`. If it's a method then it's `Test` + `StructName` + `_MethodName`

Reply:
why am i incompetent

Reply:
that's why you use gotests to generate test funcs! they never get the name wrong

Reply:
ok then explain Test_isPrereqSatisfied

Reply:
ok idk wtf is happening

Reply:
maybe it's because it's a private func so it wants to maintain the lowercase

Reply:
yeah can confirm that is the case, the more you know! LOL

Reply:
go is dumb

Reply:
at least we have consistency in naming convention